test/e2e/fixtures/webhook/webhook.go

```diff
@@ -59,7 +60,7 @@ func (s *AdmissionWebhookServer) StartTLS(t *testing.T, certFile, keyFile string
 
 	go func() {
 		err := serv.ListenAndServeTLS(certFile, keyFile)
-		if err != nil && err != http.ErrServerClosed {
+		if err != nil && errors.Is(err, http.ErrServerClosed) {
```

Comment:
This looks like a change in logic to me (from "err is **not** `http.ErrServerClosed`" to "err _is_ `http.ErrServerClosed`"). Is that intentional?

Reply:
no, its fixed now